docs/quick_tour/sqla_demo/sqla_demo/models/meta.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import engine_from_config
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.schema import MetaData
+import zope.sqlalchemy
+
+NAMING_CONVENTION = {
```

Comment:
Drop a comment with a link to http://alembic.readthedocs.org/en/latest/naming.html right above it.